### PR TITLE
  Update extension-grants.rst with example

### PR DESCRIPTION
**Pull request by FStefanni**
Hi,

just updated a doc example, which seems fine to me.
I have left classes on purpose, since it is a more modern JS syntax, and probably also this code base is moving to more modern JS. 

Regards.

Edit by @jankapunkt: Fixed issue 89, point 5, original issue 530

### CODE REVIEW

**Comment by jankapunkt** [comment]
Can someone else review this, I am not using the rst files

**Review by jorenvandeweyer — APPROVED** [review]
Looks fine, but can we merge it without the changes to `CHANGELOG.md`, `package.json` and `package-lock.json`?

**Comment by FStefanni** [comment]
Hi,

revertyed the changes on `CHANGELOG.md`, `package.json` and `package-lock.json`.

Regards

**Comment by jankapunkt** [comment]
@Uzlopak @HappyZombies @jorenvandeweyer should we include this in 4.2.0 ?

File: docs/misc/extension-grants.rst [hunk 1]
```diff
@@ -2,7 +2,51 @@
  Extension Grants
 ==================
 
-.. todo:: Describe how to implement extension grants.
+Create a subclass of ``AbstractGrantType`` and create methods `handle` and `saveToken` along with other required methods according to needs
```
Comment (Uzlopak) [inline docs/misc/extension-grants.rst:5]:
```suggestion
Create a subclass of ``AbstractGrantType`` and create methods `handle` and `saveToken` along with other required methods according to your needs:
```

**Review by Uzlopak — DISMISSED** [review]
I think it is ok so far.

**Comment by HappyZombies** [comment]
Yes it's ok so far and I'll put it for 4.2.0

@FStefanni , please update the typo and we can merge it

**Comment by FStefanni** [comment]
Done :)

**Review by jankapunkt — APPROVED** [review]

**Pull request merged**